public/components/notebooks/components/note_table.tsx

```diff
@@ -77,6 +80,13 @@ export function NoteTable(props: NoteTableProps) {
     props.fetchNotebooks();
   }, []);
 
+  useEffect(() => {
+    const url = window.location.hash.split('/')
+    if (url[url.length-1] === 'create') { 
+      createNote();
+    }
+  }, [location]);
+
```

Comment:
Nice 😄

Reply:
@derek-ho FYI future code changes (don't address for this PR)

Array.at(-1) vs arr[arr.length - 1]
The usual practice is to access [length](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/length) and calculate the index from that — for example, array[array.length - 1]. The at() method allows relative indexing, so this can be shortened to array.at(-1).